src/Stub.php

```diff
@@ -415,7 +415,12 @@ public static function constructEmptyExcept(
 
     private static function generateMock()
     {
-        return self::doGenerateMock(func_get_args());
+        $args = func_get_args();
+        if (version_compare(PHPUnitVersion::series(), '10.4', '>=') && !is_bool($args[1])) {
+            array_splice($args, 1, 0, [true]);
```

Comment:
https://github.com/sebastianbergmann/phpunit/commit/c8e2aa0c54770f1124013d2b4ecc61eebc6aa98b added a new parameter to `Generator::testDouble()`